ftl/common/builder.py

```diff
@@ -114,7 +114,10 @@ def Build(self):
     def AppendLayersIntoImage(self, lyr_imgs):
         for i, lyr_img in enumerate(lyr_imgs):
             if i == 0:
-                result_image = lyr_img
+                if hasattr(lyr_img, 'GetImage'):
```

Comment:
can you change this to 
```
try:
  result_image = lyr_img.GetImage()
except AttributeError:
  result_image = lyr_img
```

it's a little safer and wouldn't require that we change the string in `hasattr()` if the method name ever changes.

Reply:
done